clarity/enhancer/compressor.py

```diff
@@ -1,41 +1,42 @@
 """Compressor Class"""
 from __future__ import annotations
 
-from typing import Any
+from typing import Any, Final
 
 import numpy as np
 
+EPS: Final = 1e-8
+
 
 class Compressor:
     def __init__(
         self,
-        fs: int = 44100,
-        attack: int = 5,
-        release: int = 20,
-        threshold: float = 1,
+        fs: float = 44100.0,
```

Comment:
Why was `fs` changed to float? 
Perhaps, we could change this parameter to `sample_rate` like NAL-R which usually are used together.

Reply:
It's more 'correct' I think and less 'opinionated':

- Samples per second can be an arbitrary number. It just happens that standard audio formats quote sample rates as integers 16000, 44100 etc.  If we were to be passing a downsampled signal to a function we may very well have a fractional sampling rate.

- Float is more permissive: i.e. float will accept floats or ints; but int will complain if it receives a float -- even a float that happens to be roundable to an int such as 16000.0.  So my approach has been to start off with quite permissive types -- we can always choose to restrict further later. 


```
def fn_int(x_int: int):
    print(x_int)

def fn_float(x_float: float):
    print(x_float)

fn_int(1)  # OK, obviously
fn_float(1.0)  # OK, obviously
fn_int(1.0)  # <-- not OK
fn_float(1)  # <-- but also ok as float can accepts ints or floats
```

- ints can become floats in unexpected ways: e.g. ints turn into floats on division -- even if they are even -- unless you are using integer division '//' -- which is not used consistently in the repo, e.g. in torch_msbg there is "nyquist = sr / 2"  

```
In [6]: type(16000/2)
Out[6]: float
```